bin/phpactor.php

```diff
@@ -1,38 +1,37 @@
 <?php
+use Phpactor\Console\Application;
+
+use Symfony\Component\Debug\Debug;
 
 use Symfony\Component\Console\Input\ArgvInput;
```

Comment:
No empty spaces between use statements and one blank line after `<?php` (I know Phpactor does this sometimes, that's a bug!)

Reply:
No that's on me, I do have tendencies to split uses by groups. Reverting.